asv_benchmarks/benchmarks/tracking/maskers.py

```diff
@@ -0,0 +1,50 @@
+"""Benchmarks for masker objects under nilearn.maskers module."""
+
+# ruff: noqa: RUF012
+
+from ..common import Benchmark
+from ..utils import apply_mask, load
+
+
+class NiftiMaskerBenchmark(Benchmark):
+    """
+    Benchmark for applying a mask to an image using NiftiMasker with different
+    parameters.
+    """
+
+    # try different combinations of parameters for the NiftiMasker object
+    param_names = ["smoothing_fwhm", "detrend"]
```

Comment:
Out of curiosity: Is there a reason for picking these two specific parameters only?

Reply:
`detrend` is arbitrary just to test the parameterizing in asv

I did want to look into smoothing though because of this issue: https://neurostars.org/t/problem-with-smoothing-in-first-level-analysis/29553/2

But that's probably GLM-specific.

Reply:
Yeah I think that is a good reason to include smoothing here.